### PR TITLE
2206 price range

### PR DESCRIPTION
**Pull request by asogwachiuba**
What is in this PR:
The event price filter is updated for direct price input.
Screenshot_2023-04-26-09-27-45-706_com ibloov app

Screenshot_2023-04-26-16-36-50-453_com ibloov app

### CODE REVIEW

File: lib/core/services/location/location_service.dart [hunk 1]
```diff
@@ -77,6 +77,26 @@ class LocationService {
     return null;
   }
 
+  Future<bool> checkLocationPermissionStatus() async {
```
Comment (divincoder) [inline lib/core/services/location/location_service.dart:80]: Did you test this, like not accept to turn on location and see of it prompts you to?

**Review by divincoder — CHANGES_REQUESTED** [review]
Add screenshots

File: lib/core/services/location/location_service.dart (hunk 1 above)
Comment (divincoder) [reply lib/core/services/location/location_service.dart:80]: Yes, a dialog should pop up to tell the user to goto settings and update their location. Does that happen at the moment?

Comment (asogwachiuba) [reply lib/core/services/location/location_service.dart:80]: A dialog doesn't pop up but rather its written on the location tab telling them to go their settings to enable it.

Comment (divincoder) [reply lib/core/services/location/location_service.dart:80]: Yes, it should be a dialog. Why is the screenshot above not reflecting this scenario

Comment (asogwachiuba) [reply lib/core/services/location/location_service.dart:80]: Location was chosen for this one. I will make the update and share a screenshot of it here too.

**Pull request merged**